python/tvm/relay/frontend/pytorch.py

```diff
@@ -2060,10 +2060,57 @@ def _get_convert_map(prelude):
         "aten::type_as"                         : _type_as(),
         "aten::gather"                          : _gather(),
         "aten::index_select"                    : _select(),
+        "torchvision::nms"                      : _nms(prelude),
     }
     return convert_map
 
 
+def _nms(prelude):
```

Comment:
Following other op converters, please put the definition of `_nms` before `_get_convert_map` (after `_meshgrid`).